_posts/2023-10-24-filesystem-integrity-elytron-tool.adoc

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: 'Adding Support for Integrity Checking to an Existing 
+Filesystem Realm'
+date: 2023-10-24
+tags: filesystem integrity filesystem-realm
+synopsis: How to add Integrity checking to an existing filesystem realm using the Elytron tool. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+WildFly 27 included the ability to add integrity on FileSystem realms. This uses an asymmetric keyPair that has a private and public key to secure the filesystem realm. In the filesystem realm, each identity will be signed using the private key. This makes it possible to verify that identities in the realm haven't been tampered with.
+
+toc::[]
+
+https://wildfly-security.github.io/wildfly-elytron/blog/filesystem-integrity/[A previous blog post] demonstrates how to create a filesystem realm and secure it using the elytron subsystem. This blog post will demonstrate how to take an existing filesystem realm and add integrity checking to it using the WildFly Elytron Tool. In this post we will go through an example of converting an unencrypted filesystem realm to one with integrity enabled and then we will try accessing a web application that is secured with the new filesystem realm.
```

Comment:
Fixed! 
I changed it to "demonstrates how to enable integrity support on a filesystem realm using the Elytron subsystem" since I was talking about this (https://wildfly-security.github.io/wildfly-elytron/blog/filesystem-integrity/) blog.